crds.yaml

```diff
@@ -0,0 +1,287 @@
+
```

Comment:
Is this file here by mistake? I guess when you were playing with the script to generate the manifests

Reply:
No, the helm template doesn't contain the crds hence https://github.com/utilitywarehouse/cert-manager-manifests/pull/21/files#diff-0ff681169182e8c00778fa8e1008b76170e4ac423608c3b5fbb9fdc897ea1709R5. The crds are on a separate kustomize base otherwise the patches to remove helm labels would get more complicated.

Reply:
nvm, will fix